src/main/java/seedu/address/logic/parser/UnassignProjectCommandParser.java

```diff
@@ -0,0 +1,48 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_EMPLOYEE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PROJECT;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.UnassignProjectCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new UnassignProjectCommand object
+ */
+public class UnassignProjectCommandParser implements Parser<UnassignProjectCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the UnassignProjectCommand
+     * and returns an UnassignProjectCommand object for execution.
+     *
+     * @throws ParseException
+     */
+    public UnassignProjectCommand parse(String args) throws ParseException {
+        requireNonNull(args);
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args,
+                PREFIX_PROJECT, PREFIX_EMPLOYEE);
+
+        if (!arePrefixesPresent(argMultimap, PREFIX_PROJECT, PREFIX_EMPLOYEE)) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                                     UnassignProjectCommand.MESSAGE_USAGE));
+        }
```

Comment:
We should check whether the user has provided duplicate prefixes using `argMultimap.verifyNoDuplicatePrefixesFor(PREFIX)` to prevent undefined behaviour

Reply:
Done.